server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/http/SessionRoutesTest.scala

```diff
@@ -61,18 +61,18 @@ class SessionRoutesTest extends AnyFlatSpec with BeforeAndAfter with Matchers {
     when(mockedAuthFilter.authenticate(any()))
       .thenReturn(Mono.just(mockedSession))
 
-    sessionSupplier = spy(new SessionSupplier())
-    val jmapRoutes: Set[JMAPRoutes] = Set(new SessionRoutes(
-      sessionSupplier = sessionSupplier,
-      authFilter = mockedAuthFilter))
+    val sessionRoutes = new SessionRoutes(
+      sessionSupplier = new SessionSupplier(),
+      authenticator = mockedAuthFilter)
     jmapServer = new JMAPServer(
       TEST_CONFIGURATION,
-      jmapRoutes.asJava)
+      Set(new JMAPRoutesHandler(Version.RFC8621, sessionRoutes)).asJava,
+      new VersionParser(Set(Version.RFC8621).asJava))
     jmapServer.start()
 
     RestAssured.requestSpecification = new RequestSpecBuilder()
       .setContentType(ContentType.JSON)
-      .setAccept(ContentType.JSON)
+      .addHeader(ACCEPT.toString, s"application/json; jmapVersion=${Version.RFC8621.asString}")
```

Comment:
Is the word `s` are wrong typing?

Reply:
no, it is a shortcut in scala https://docs.scala-lang.org/overviews/core/string-interpolation.html